tockloader-lib/src/tabs/mod.rs

```diff
@@ -0,0 +1,6 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright OXIDOS AUTOMOTIVE 2024.
+
+pub mod app_tab;
+pub mod tab;
```

Comment:
The two mod names aren't that distinct. It's unclear what the difference between them.